### DISCUSSION TITLE
智能隱形眼鏡：「近在眼前」的電腦螢幕？

### DISCUSSION THREAD

**Discussion by HuangHsinTzu** [1101-GS4719-C/Python-Community-GS4719-C#750] [科技新知]
我們常常在電影裡看到主角帶著超帥的智能眼鏡
可以協助主角搜尋目標，導航甚至是和其他人視訊通話的功能。
近年來Mojo Vision公司研發出一種智能隱形眼鏡，
能覆蓋眼睛眼白部分的隱形眼鏡鏡片，能矯正佩戴者的視力，同時也附帶一個微型LED顯示器，智能感應器和固態電池。
除了實現電影裡的智能眼鏡的功能，
研究人員正在努力開發更多醫療相關的功能，
像是蒐集糖尿病患者的血糖資訊、診斷及治療眼部疾病甚至是檢查有無特定的癌症指標等
雖然智能眼鏡有很大的發展潛力，
但仍存在像是供電無法長久或是隱私等問題需克服
(有人擔心有心人士會利用此智能眼鏡進行偷拍)
參考文章: https://www.bbc.com/zhongwen/trad/science-61550250

**Comment by manipulateed** [comment]
想不到電影場景的令人嚮往的科技正在一步步被執行中，真的非常興奮，但同時對於安全或是各個層面的問題還有待討論與限制。

**Comment by williamyen0623** [comment]
希望我能趕快體驗到 Link Start

**Comment by BlahFish1006** [comment]
沒想到韓劇《阿爾罕布拉宮的回憶》中，透過隱形眼鏡實現虛擬擴增實境遊戲的體驗正在現實中被慢慢實現，真的太令人期待了!
隱形眼鏡比起其他穿戴式裝置，需要滿足更嚴苛的條件 (比如需要在更輕更小更薄的微型晶片上具有一定時長的續航力等等)，好好奇他們到底是如何解決這個難題的喔030

**Comment by Pietra1226** [comment]
太厲害了，真的是越來越方便，也越來越享受了

**Comment by delia07152** [comment]
聽起來很棒欸，能同時達到醫療以及娛樂的效果。

**Comment by moonnight3503** [comment]
太酷了！在眼前可以直接顯示字跟樂譜，我也想要有一副智能隱形眼睛🥺

**Comment by dabbie109303501** [comment]
太酷了，科技的進步漸漸把科幻電影中描述的場景重現在生活中

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
聽起來就好酷 但不知道產品的發布會不會有些法律和道德的問題

**Comment by CPCpyt** [comment]
我覺得功能非常前衛，概念也很吸引人。
針對個人的確是能有很多的幫助，
不過涉及隱私資料的部分我覺得有很大的商議空間。

**Comment by peggy0329** [comment]
我已經無法想像現在科技可以把智能用在隱形眼鏡上了
雖然功能很完善且方便
但隱私問題還有像是把需要用到電源的東西放在我的眼睛上就覺得怕怕的
目前這個產品帶給人的效益感覺小於成本
不知道會不會有人買單

**Comment by ruthchen1020** [comment]
很期待，但也有健康上的疑慮。如果我不想每天帶隱眼怎麼辦🤣
現在都可以有2奈米的加工技術了，我覺得這個產品在不久的將來會實現。

**Comment by StevenWanglolz** [comment]
感覺如果真的能達到智能眼鏡的話，可能會帶來很多安全上的疑慮，各種偷拍或是機密資料可能會外洩

**Comment by wenchi9023** [comment]
科幻電影場景逐步在現實生活出現真的讓人興奮!
但是光看文章的描述就讓我感到可怕，除了隱私問題外，
人類只有一雙眼睛，若這項產品稍有不慎便可能帶給使用者無法挽回的後果

**Comment by Evan0979** [comment]
好像科幻電影裡面的特務啊!讓我看得一整個人都興奮起來了!

**Comment by dinosaur1009** [comment]
這樣的科技真的很酷！但是我覺得離要讓大眾普遍接受應該還是存在一定難度

**Comment by orianna901001** [comment]
我感覺科技真的要步步走入人類的世界了，我覺得如果要讓大眾接受的話，或許可以創立一個園區，讓大家可以進到裡面然後互動，看大眾買不買單，有點類似迪士尼、環球的各個園區。

**Comment by quan0715** [comment]
一個能覆蓋整個眼白的隱形眼鏡會不會適配性很低啊，我自己隱形眼鏡就帶到快死了，再說如果是這樣的話為何不直接生產眼鏡即可，像文中提到那種大型的侵入式物件，不免讓我擔心他的市場價值與未來延展性

**Comment by harrycoconut** [comment]
太酷了！這讓我也想到一部影集叫黑鏡，其中一集的主角就擁有類似的技術，發生的任何事他都能紀錄下來並重播，是個非常酷的反烏托邦設定！在這邊就先不爆雷了，這篇的作者可以去看看這部影集喔～😌

**Comment by Jamesmoneya** [comment]
有去參加一個密室逃脫，它裡面有一款道具就是類似的眼鏡，我覺得最大的困難點就是要防止暈眩，不然戴上去真的會超級暈的，沒辦法帶超過1分鐘
